WordPress/Classes/Models/BasePost.swift

```diff
@@ -34,6 +34,21 @@ extension BasePost {
         case trash = "trash"
         case deleted = "deleted" // Returned by wpcom REST API when a post is permanently deleted.
     }
+
+    static let pathForDisplayImageKeyPath = "pathForDisplayImage"
+    var pathForDisplayImage: String? {
```

Comment:
Is there a reason why we'd only want to do it after an version update? 

Reliably detecting that is a PITA, doing changes to `posts` seems like it could be potentially dangerous when mixed with Core Data issues and seems like a recipe for some race conditions that will haunt us forever.

This is fairly straightfoward bit of string comparisons, which, unless the posts are HUGE and/or have thousands of images, shouldn't take too long — doing it on-demand, when a failed image is loaded into the editor seems like a reasonable enough thing for me.